spec/generators/rolify/rolify_activerecord_generator_spec.rb

```diff
@@ -53,24 +53,25 @@ class User < ActiveRecord::Base
           should contain "class Role < ApplicationRecord"
         end
       end
-      it { should contain "has_and_belongs_to_many :users, :join_table => :users_roles" }
+      it { should contain "  has_and_belongs_to_many :users, :join_table => :users_roles" }
       it do
         if Rails::VERSION::MAJOR < 5
-          should contain "belongs_to :resource,\n"
-                          "           :polymorphic => true"
+          should contain "  belongs_to :resource,\n"
+                          "             :polymorphic => true"
         else
-          should contain "belongs_to :resource,\n"
-                          "           :polymorphic => true,\n"
-                          "           :optional => true"
+          should contain "  belongs_to :resource,\n"
```

Comment:
I'm not a big fan of these extra spaces, don't get the added value here

Reply:
I removed this additional spaces